### ISSUE TITLE
Offer an "Implement with Mixin" code action when adding an interface to a type

### ISSUE THREAD

**Issue by pgenfer** [pgenfer/mixinSharp#14]
**Remark**: Issue #11 was split into two parts for easier administration. This is the second part.

Say I have the following mixin
```C#
public class NameMixin: INamed
{
    public string Name { get; set; }
}
```
and ```INamed``` is defined as follows
```C#
public interface INamed
{
    string Name { get; set; } = string.Empty;
}
```
it would be very convenient to offer an _Implement with NamedMixin_ code action when adding the ```INamed``` interface to a type not having an implementation.

Thanks to @aluanhaddad for suggesting this feature.

**Comment by aluanhaddad** [comment]
Thanks for implementing this. I love the feature!

**Comment by pgenfer** [comment]
Implemented in V1.4
